eth/eth.go

```diff
@@ -42,30 +47,44 @@ func NewBlock(blockStore store.BlockStoreReader, metrics Metrics) *Block {
 	}
 }
 
-func (e *Block) GetBlockByNumber(id BlockID, inclTx bool) (map[string]any, error) {
+var signerConfig = &params.ChainConfig{
+	LondonBlock: new(big.Int),
```

Comment:
Interesting. I can remove it for now. My hunch is that the tests will fail when we include tests for L1 deposit txs, since right now the code is effectively untested. Thanks for catching this.